### DISCUSSION TITLE
Add a Dynamic Tag to show a future Date

### DISCUSSION THREAD

**Discussion by katesha32** [elementor/elementor#25364] [Feature Request]
### Prerequisites

- [X] I have searched for similar feature requests in both open and closed discussions and cannot find an existing request.
- [X] I have verified that the feature is still missing from the latest stable versions of Elementor or Elementor Pro.

### What problem is your feature request going to solve? Please describe..

Date which shows tomorrow constantly 

### Describe the solution you'd like.

On main screen of my landing page, some header will show the dynamic date which constantly shows the date of the next day

### Agreement

- [X] I agree that my request may be closed without action if it doesn't meet all the requirements.

**Comment by m1ga** [comment]
https://developers.elementor.com/docs/getting-started/first-addon/
and add the code from the "tomorrow" example in https://www.w3schools.com/php/php_date.asp (search for it) 
```php
$d=strtotime("tomorrow");
echo date("Y-m-d h:i:sa", $d) ;
```

into the `render()` function.


Or you create a WP shortcut and use that:
https://www.hostinger.com/tutorials/create-a-shortcode-in-wordpress
with the same date code.

**Comment by nicholaszein** [comment]
Hi!

You can achieve this functionality by using the [Countdown widget](https://elementor.com/help/countdown-widget-pro/) (Pro) with the Type of Evergreen timer. Then you can set the timer to 24h, and customizer your timer.

Otherwise you can follow @m1ga's suggestion to create your own custom display.  

You can also create your own Dynamic Tag for this functionality by following the [instructions in the Developers Documentation](https://developers.elementor.com/docs/dynamic-tags/) and applying @m1ga's logic above.

We currently have no plans to implement this feature. However, we will consider it if there is enough demand (votes) from other users.

Thank you for your contribution! 🙏 

Kind regards.
